tests/src/test/scala/cats/tests/ApplicativeErrorSuite.scala

```diff
@@ -35,5 +35,31 @@ class ApplicativeErrorSuite extends CatsSuite {
     failed.recoverWith { case _ => Some(7) } should === (Some(7))
   }
 
+  {
+    final case class OptionWrapper[A](option: Option[A])
```

Comment:
Thanks for the quick fix. Just FYI (not saying we should introduce another dependency in cats), with https://github.com/estatico/scala-newtype, all these can be written as
```scala
@newtype case class OptionWrapper[A](option: Option[A])
object OptionWrapper {
    implicit def ae[E](implicit ev: ApplicativeError[Option, E]): ApplicativeError[OptionWrapper, E] = ev.asInstanceOf
}
```

Reply:
thanks, yes, I would love to use it 😄 , very compact indeed. May be in the near future ... 😃